src/runtime-rs/crates/shim-ctl/src/main.rs

```diff
@@ -0,0 +1,40 @@
+use common::{
+    message::Message,
+    types::{ContainerConfig, Request},
+};
+use runtimes::RuntimeHandlerManager;
+use tokio::sync::mpsc::channel;
+
+const MESSAGE_BUFFER_SIZE: usize = 8;
+
+async fn real_main() {
+	let (sender, _receiver) = channel::<Message>(MESSAGE_BUFFER_SIZE);
+	let manager = RuntimeHandlerManager::new ("xxx", sender).await.unwrap();
+
+    let req = Request::CreateContainer(ContainerConfig {
+        container_id: "xxx".to_owned(),
+        bundle: ".".to_owned(),
+        rootfs_mounts: Vec::new(),
+        terminal: false,
+        options: None,
+        stdin: None,
+        stdout: None,
+        stderr: None,
+    });
+
+    let result = manager.handler_message(req).await;
+
+    println!("{:?}", result);
```

Comment:
Delete this line.

Reply:
Done, although it seems worth a mention that even the actual shim uses `println!()` at this level.